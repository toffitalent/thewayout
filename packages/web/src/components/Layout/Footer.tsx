import { Container, Text } from '@disruptive-labs/ui';
import Link from 'next/link';
import styles from './Footer.module.scss';

export const Footer = () => (
  <footer className={styles.footer}>
    <Container>
      <Text color="grey.500" fontWeight="700">
        The Way Out, Inc. &copy; {new Date().getFullYear()}
        <Link href="/terms/" passHref>
          Terms
        </Link>
        <Link href="/privacy/" passHref>
          Privacy
        </Link>
      </Text>
    </Container>
  </footer>
);
