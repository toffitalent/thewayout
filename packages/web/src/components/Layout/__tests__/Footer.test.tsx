import { render } from '@test';
import { Footer } from '../Footer';

describe('Footer', () => {
  test('renders correctly', () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
