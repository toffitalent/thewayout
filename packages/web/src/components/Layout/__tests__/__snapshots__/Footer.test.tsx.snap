// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    <div
      class="container"
    >
      <p
        class="text color-grey-500 fontWeight-700"
      >
        The Way Out, Inc. Â© 
        2024
        <a
          href="/terms"
        >
          Terms
        </a>
        <a
          href="/privacy"
        >
          Privacy
        </a>
      </p>
    </div>
  </footer>
</div>
`;
